tfctl/uninstall.go

```diff
@@ -0,0 +1,101 @@
+package tfctl
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/fluxcd/pkg/ssa"
+	"github.com/theckman/yacspin"
+	appsv1 "k8s.io/api/apps/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// Uninstall removes the tf-controller resources from the cluster.
+func (c *CLI) Uninstall(out io.Writer) (retErr error) {
+	ctx := context.Background()
+
+	var deployment appsv1.Deployment
+	if retErr := c.client.Get(ctx, types.NamespacedName{
+		Namespace: c.namespace,
+		Name:      "tf-controller",
+	}, &deployment); retErr != nil {
+		if apierrors.IsNotFound(retErr) {
+			fmt.Fprintf(out, "tf-controller not found.\n")
```

Comment:
You're not formatting the text here, so you could use [fmt.Fprintln](https://pkg.go.dev/fmt#Fprintln).